src/algorithms/graphs/bfs.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::VecDeque;
+
+#[allow(dead_code)]
+pub fn bfs(adj: &Vec<Vec<usize>>, vis: &mut Vec<bool>, s: usize) {
+    let mut q = VecDeque::new();
+    q.push_back(s);
+    vis[s] = true;
+    while !q.is_empty() {
+        let u = q.pop_front().unwrap();
+        vis[u] = true;
+        for i in 0..adj[u].len() {
+            let v = adj[u][i];
+            if !vis[v] {
+                q.push_back(v);
+            }
+        }
```

Comment:
@Segmentree could you change this for as in the dfs?

Reply:
@rbruno95 I think the correct way o say is `for how in the dfs` but yes no problem

Reply:
@Segmentree I think I wrote it correctly, I've just checked it using DeepL